_includes/v22.2/known-limitations/cdc-transformations.md

```diff
@@ -0,0 +1,9 @@
+- It is necessary to pass the [`schema_change_policy='stop'`](create-changefeed.html#schema-policy) option in the changefeed creation statement when using the {% if page.name == "cdc-transformations.md" %} CDC transformations {% else %} [CDC transformations](cdc-transformations.html){% endif %} format. [Tracking GitHub issue](https://github.com/cockroachdb/cockroach/issues/85143).
+- You can only apply CDC transformations on a single table in each statement.
+- Some stable functions, notably functions that return MVCC timestamps, are overridden to return the MVCC timestamp of the event.
+- You cannot [alter](alter-changefeed.html) a changefeed that uses CDC transformations. [Tracking GitHub issue](https://github.com/cockroachdb/cockroach/issues/83033)
```

Comment:
same nit: consider `Currently, ...`

Reply:
We don't want to add forward-looking references per style guide. I think this still applies to both of these. Yes, this should be fixed, but we can't guarantee that. Probably better to just state what is the current state of the feature.